app/assets/javascripts/charts.js

```diff
@@ -240,7 +240,7 @@ function reset_zoom(item){
 
 function showTooltip(pos, item, formater) {
   var content = formater(item);
-  $('<div id="flot-tooltip">').text(content).css({
+  $('<div id="flot-tooltip">').html(content).css({
```

Comment:
Would it maybe be better to remove the `<b>` tags and keep this as text? that way we don't open the possibility of xss here.

Reply:
I guess it would :+1: